### DISCUSSION TITLE
Reduce Maui app (apk) size

### DISCUSSION THREAD

**Discussion by SebastianKleu** [dotnet/maui#6000] [Q&A]
In Xamarin, I was able to reduce the apk size by setting "AndroidSupportedAbis" to only create a published build for a specific architecture (ex. "armeabo-i-v7a"). I was also able to set "AndroidLinkMode" to "SdkOnly" to decrease the application size. Is this at all possible in Maui?

**Comment by Keivan-Mousavi** [comment]
Hi
Please check this link 
[https://devblogs.microsoft.com/dotnet/announcing-net-maui-preview-9/](url)

**Reply by MarshalOfficial** [reply]
this link is 404 not found.

**Reply by MuratTopcu** [reply]
> this link is 404 not found.
You can copy and paste the link above or I'll leave the working link below :)
[https://devblogs.microsoft.com/dotnet/announcing-net-maui-preview-9/](https://devblogs.microsoft.com/dotnet/announcing-net-maui-preview-9/)
